flow-typed/final-form.js

```diff
@@ -27,3 +27,7 @@ declare type FinalFormMeta = {
   valid?: boolean,
   visited?: boolean,
 }
+
+declare type FinalFormValidateType = 'touched' |
```

Comment:
Are you sure, that we need this type everywhere else, except JInputField component?

Reply:
I think we need this functionality in all fields